app/controllers/junior_stories_controller.rb

```diff
@@ -33,4 +37,41 @@ def junior_story_params
       :remote, :tech_team_size, :company_size, :company_age, :person_of_colour, :other,
       :publishing_consent, :freelancer)
   end
+
+  def allowed_params
```

Comment:
Nope, the difference is in the `require(:junior_story)`. Do we really need that above?